### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by testwill**
**Please provide a description of this PR:**

### CODE REVIEW

**Comment by ericvn** [comment]
You will need to run `make gen` and check in the changed file.

**Review by howardjohn — APPROVED** [review]

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ message SidecarInjectorConfig {
   // Enables sidecar auto-injection in namespaces by default.
   google.protobuf.BoolValue enableNamespacesByDefault = 2;
 
-  // Setting this to `IfNeeded` will result in the sidecar injector being run again if additonal mutations occur. Default: Never
+  // Setting this to `IfNeeded` will result in the sidecar injector being run again if additional mutations occur. Default: Never
```
Comment (hanxiaop) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto:1041]: Please run make gen to generate the operator files.

**Review by hanxiaop — APPROVED** [review]

**Review by morvencao — APPROVED** [review]

**Pull request merged**